File: executable/node/horizontal-histogram.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var y = Math.random() * 50

var data = [
  {
    y: y, 
    type: "histogram"
  }
];

var graph_options = {filename: "horizontal-histogram", fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: executable/r/horizontal-histogram.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

y = rnorm(500)

data <- list(
  list(
    y = y, 
    type = "histogram"
  )
)

response <- p$plotly(data, kwargs=list(filename="horizontal-histogram", fileopt="overwrite"))
url <- response$url
filename <- response$filename